docs/api/ethereum.md

```diff
@@ -1644,14 +1650,21 @@ const mutation = await ledger.transferAsset(recipe);
 ## Ledger abilities
 
 Ledger abilities represent account-level permissions. For optimization reasons abilities are managed as bitfields for that reason enums are values of 2**n.
+We have two categories of abilities. General and super. General abilities are abilities that can not change other account's abilities and super abilities can.
+This categorization is for safety purposes since revoking your own super ability can lead to unintentional loss of control. 
 
-**Options:**
```

Comment:
Let's merge that into one table but add a column `category`.

Reply:
The point is to have it separate to show that this abilities are different and demand more caution. If they are in the same table we lower that difference...